app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>user profil</h1>
\ No newline at end of file
+<h1>The users profil</h1>
+<div>
+    <div class='user-container'>
+      <%= image_tag @user.photo, :class => 'user-photo', alt: 'User photo' %>
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.PostsCounter.nil? ? 0 : @user.user_posts %></p>

```

Comment:
- [ ] This line does not show the number of posts, it shows a post object. I recommend correcting that for a better user experience.

<img width="479" alt="image" src="https://user-images.githubusercontent.com/77190203/188264817-fe812fb0-bfe8-47c7-b401-fb708b607215.png">